e2e/forkproxy/app/app.go

```diff
@@ -0,0 +1,245 @@
+package app
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os/exec"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+
+	"golang.org/x/sync/errgroup"
+)
+
+type Config struct {
+	ListenAddr    string
+	ChainID       uint64
+	LoadState     string
+	BlockTimeSecs uint64
```

Comment:
Does it make sense for this to be `time.Duration`?

Reply:
I'm sticking to the anvil API for consistency